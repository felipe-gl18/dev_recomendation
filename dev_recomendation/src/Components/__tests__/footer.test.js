import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../footerComponent';

it('snapshot footer', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
})