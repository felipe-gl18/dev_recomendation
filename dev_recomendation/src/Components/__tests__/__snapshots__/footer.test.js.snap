// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot footer 1`] = `
<footer>
  <p>
     Â© felipe-gl18
  </p>
</footer>
`;
